packages/client/src/views/SearchResult/SearchResult.test.tsx

```diff
@@ -410,13 +410,15 @@ describe('SearchResult tests', () => {
           operationName: null,
           query: SEARCH_EVENTS,
           variables: {
-            locationIds: null,
-            sort: 'DESC',
-            trackingId: 'DW0UTHR',
-            nationalId: '',
-            registrationNumber: '',
-            contactNumber: '',
-            name: ''
+            advanceSearchParameters: {
+              declarationLocationId: '2a83cf14-b959-47f4-8097-f75a75d1867f',
```

Comment:
Previously we had `locationIds: null` in the tests. But now we are giving it a particular locationId. Do we need to care about this?

Reply:
Already tested and it's working. That's why didn't add or modify the test.